### PR TITLE
Re-generated Network for August release

### PR DESCRIPTION
**Pull request by anton-evseev**
## Description

Based on: https://github.com/Azure/azure-rest-api-specs/pull/3635

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj [hunk 1]
```diff
@@ -7,11 +7,15 @@
 		<PackageId>Microsoft.Azure.Management.Network</PackageId>
 		<Description>Provides management capabilities for Network services.</Description>
 		<AssemblyName>Microsoft.Azure.Management.Network</AssemblyName>
-		<Version>19.2.0-preview</Version>
+		<Version>20.0.0-preview</Version>
```
Comment (dsgouda) [inline src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj]: @MikhailTryakhov any word on when you are planning to release a stable version here? Any reason why you plan to do a major version bump here

File: src/SDKs/Network/Network.Tests/Tests/LoadBalancerTests.cs [hunk 2]
```diff
@@ -1435,141 +1435,5 @@ public void LoadBalancerNatPoolTest()
                 networkManagementClient.PublicIPAddresses.Delete(resourceGroupName, lbPublicIpName);
              }
         }
-
-        [Fact(Skip="Disable tests")]
```
Comment (dsgouda) [inline src/SDKs/Network/Network.Tests/Tests/LoadBalancerTests.cs]: Why are these being deleted? Was this service deprecated?

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks great for the most part, need some questions answered

File: src/SDKs/Network/Network.Tests/Tests/LoadBalancerTests.cs (hunk 2 above)
Comment (number213) [reply src/SDKs/Network/Network.Tests/Tests/LoadBalancerTests.cs]: OutboundNatRules were never made public
https://github.com/Azure/azure-rest-api-specs/pull/3592

Comment (number213) [reply src/SDKs/Network/Network.Tests/Tests/LoadBalancerTests.cs]: Specific comment: https://github.com/Azure/azure-rest-api-specs/pull/3592#discussion_r210762102

File: src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj (hunk 1 above)
Comment (MikhailTryakhov) [reply src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj]: @dsgouda  I think we need publish stable version when all know issues will be fixed. now we have one in brooklyn team. let's I test all swagger issues we have (in optional jobs for example), fix them and next Ignite version will be stable

Comment (shahabhijeet) [reply src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj]: if you are in preview you cannot bump up major version

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

**Comment by number213** [comment]
@shahabhijeet @dsgouda set file version to `19.3.0`, left assembly version as `19.0.2.0` (I remember in previous PR I was asked to leave it as is)

**Review by dsgouda — APPROVED** [review]
LGTM

**Comment by dsgouda** [comment]
@shahabhijeet any additional feedback?

**Comment by number213** [comment]
I'll update this PR once LB changes (https://github.com/Azure/azure-rest-api-specs/pull/3697) are merged in `master`. That way we'll avoid breaking changes in September release

**Comment by dsgouda** [comment]
Closing PR per @MikhailTryakhov 's comment

**Comment by dsgouda** [comment]
@number213 Feel free to open a new PR when you are ready, I'll be happy to take a look

**Pull request closed without merging**